index.html

```diff
@@ -1,12 +1,62 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+<<<<<<< HEAD
+
     <title>Whether Weather</title>
+    <link rel="icon" type="image/png" href="#">
+
+    <link rel="stylesheet" href="assets/reset.css">
+
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="assets/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Baloo+Thambi|Open+Sans&display=swap" rel="stylesheet">
+
+=======
+    <title>Whether Weather</title>
+>>>>>>> 6191de160f1997becdce6bbb0219a72577de12e8
 </head>
+
 <body>
-    
+
+    <div class="jumbotron jumbotron-fluid">
+        <div class="container">
+            <div class="row">
+                <div class="col-xl-12 col-lg-12 col-md-12 col-sm-12 col-xs-12">
+                    <h1 class="display-4">Whether Weather</h1>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-xl-12 col-lg-12 col-md-12 col-sm-12 col-xs-12">
+                    <p class="lead">Search Local Events and Check if the Weather is Going to Cooperate.</p>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-xl-12 col-lg-12 col-md-12 col-sm-12 col-xs-12">
+                    <form class="form-inline my-2 my-lg-0">
+                        <input class="form-control mr-sm-2" type="search" placeholder="Zipcode" aria-label="Zipcode">
+                        <button class="btn btn-outline-light my-2 my-sm-0" type="submit">Search</button>
```

Comment:
change this line to :
<input type="submit" class="btn btn-outline-light my-2 my-sm-0" name="Search" id="submit-button">

Reply:
<input type="submit" class="btn btn-outline-light my-2 my-sm-0" name="Search" id="submit-button">

Reply:
//--<input type="submit" class="btn btn-outline-light my-2 my-sm-0" name="Search" id="submit-button">--//